### DISCUSSION TITLE
Option to switch the header view - dates with time

### DISCUSSION THREAD

**Discussion by sudheerkarumanchi** [hoangnm/react-native-week-view#300] [General]
Do we have any option to switch or replace the header view from dates to time and time with dates? let us know the thoughts here

**Comment by pdpino** [comment]
Do you mean this?

```
      10:00   11:00   12:00   13:00
Mon
Tue
Wed
...
```

There is no option to do that

**Reply by sudheerkarumanchi** [reply]
Ok, thanks
